src/components/victory-errorbar/helpers/borders.js

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from "react";
+
+export default class Borders extends React.Component {
```

Comment:
Sorry for the confusion. I was expecting helper methods for building up the lines inside of the `ErrorBar` component instead of separate components. This works too, but I prefer to keep as few components that actually render svg as possible, since I have to replace them all for react native implementation